dcpy/test/utils/test_versions.py

```diff
@@ -22,6 +23,40 @@ def test_parsing_invalid_version(self):
         with self.assertRaises(Exception):
             versions.parse("20231212")
 
+    def test_sort_valid_versions(self):
+        for version_list, sorted in [
+            [
+                [
+                    versions.MajorMinor(year=23, major=2),
+                    versions.MajorMinor(year=23, major=1),
+                ],
+                [
+                    versions.MajorMinor(year=23, major=1),
+                    versions.MajorMinor(year=23, major=2),
+                ],
+            ],
+            [
+                [
+                    versions.Date(year=23, month=1, day=2),
+                    versions.FirstOfMonth(year=23, month=1),
```

Comment:
Can you include a few more test cases in this list? I'd like to make sure that first of months correctly sorts both before and after `Date`s in between them (and vice-versa)

Reply:
added!